src/databases/PlainText/avtPlainTextFileFormat.C

```diff
@@ -34,6 +34,16 @@
 using     std::string;
 using     std::vector;
 
+string
+TrimLeadingandTrailingWhitespace(string str)
+{
+    auto start = str.find_first_not_of(" \t");
+    if (start == string::npos)
+        return "";
```

Comment:
I am not sure we want to return an empty string here *ever*. I know the old code could maybe have done that but lemme suggest...

```suggest
if (start == string::npos || end-start <= 0)
{
    static size_t cnt = 0;
    char tmp[16];
    snprintf(tmp, sizeof(tmp), "unknown_%zu", cnt++);
    return tmp;
}
```

Reply:
I'll change this